deep_qa/data/instances/mc_tuples_with_background_instance.py

```diff
@@ -0,0 +1,358 @@
+from collections import defaultdict
+from typing import Dict, List
+
+import numpy as np
+from overrides import overrides
+
+from .instance import TextInstance, IndexedInstance
+from ..data_indexer import DataIndexer
+
+
+class TextTuple:
+    def __init__(self, subject: str, verb: str, objects: List[str],
+                 context: str=None, time: str=None, loc: str=None, source: str=None):
+        self.subject = subject
+        self.verb = verb
+        self.objects = objects
+        self.context = context
+        self.time = time
+        self.location = loc
+        self.source = source
+
+    def __str__(self):
+        # NOTE: not currently displaying locaton, time, and source info
+        return "TextTuple( [\n\tsubj: {0} \n\tverb: {1} \n\tobjects: {2} \n\tcontext: {3}\n] )"
```

Comment:
haha -- nope, didn't actually pass them :)  yay me!
if it's ok to leave it like this, I personally find this much more readable than the other way....
(but I'll, you know, pass in stuff)